pages/verify-certification-score.vue

```diff
@@ -0,0 +1,241 @@
+<template>
+  <div class="page competences">
+    <main class="page-body">
+      <div class="container lg certificate-verification-page">
+        <h1 class="certificate-verification-page__title">{{ title }}</h1>
+        <form
+          class="certificate-verification-page__form"
+          novalidate
+          @submit="submit"
+        >
+          <div class="certificate-verification-page__form__input-field">
```

Comment:
Je crois qu'on a pas besoin d'aller aussi loin. Et reste à : Block + element + modifier. Ici on a 2 éléments.
Je suggère : 
```suggestion
          <div class="certificate-verification-page__input-field">
```
On est dans le block `certificate-verification-page` l'élement est `input-field` et on a pas de modifier